### PR TITLE
[CS0612] `QueryEventsSql` Warning Disable

### PR DESCRIPTION
**Pull request by eaba**
## Checklist

For significant changes, please ensure that the following have been completed (delete if not relevant):

* [x] This change follows the [Akka.NET API Compatibility Guidelines](https://getakka.net/community/contributing/api-changes-compatibility.html).
* [x] This change follows the [Akka.NET Wire Compatibility Guidelines](https://getakka.net/community/contributing/wire-compatibility.html).

### CODE REVIEW

File: src/contrib/persistence/Akka.Persistence.Sql.Common/Journal/QueryExecutor.cs [hunk 1]
```diff
@@ -502,10 +502,12 @@ public abstract class AbstractQueryExecutor : IJournalQueryExecutor
                     @SerializerId
                 )";
 
+#pragma warning disable CS0612 // Type or member is obsolete
```
Comment (Aaronontheweb) [inline src/contrib/persistence/Akka.Persistence.Sql.Common/Journal/QueryExecutor.cs]: Why is this `Obsolete`?

**Review by Aaronontheweb — COMMENTED** [review]
When disabling an `Obsolete` warning @eaba , we should reference the Obsoletion message and explain why disabling the warning is the right call, versus doing something else (like removing the method.)

File: src/contrib/persistence/Akka.Persistence.Sql.Common/Journal/QueryExecutor.cs (hunk 1 above)
Comment (eaba) [reply src/contrib/persistence/Akka.Persistence.Sql.Common/Journal/QueryExecutor.cs]: maybe is for this I don't know @Aaronontheweb 

https://github.com/akkadotnet/akka.net/blob/9676cc18f29ce81730d43fc019c9413f0e690533/src/contrib/persistence/Akka.Persistence.Sql.Common/Journal/QueryExecutor.cs#L563-L564

https://github.com/akkadotnet/akka.net/blob/9676cc18f29ce81730d43fc019c9413f0e690533/src/contrib/persistence/Akka.Persistence.Sql.Common/Journal/QueryExecutor.cs#L472-L477

Comment (Aaronontheweb) [reply src/contrib/persistence/Akka.Persistence.Sql.Common/Journal/QueryExecutor.cs]: @eaba is that field used at all?

Comment (eaba) [reply src/contrib/persistence/Akka.Persistence.Sql.Common/Journal/QueryExecutor.cs]: Aah! I have checked, it is not used. I want to remove it!!

**Review by Aaronontheweb — APPROVED** [review]
LGTM

**Pull request merged**